pkg/origin/convert.go

```diff
@@ -0,0 +1,45 @@
+package origin
+
+import (
+	"github.com/openshift/api/authorization/v1"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+)
+
+//ConvertAPIPolicyRulesToRBACPolicyRules -  Convert Policy Rules.
+func ConvertAPIPolicyRulesToRBACPolicyRules(in []v1.PolicyRule) []rbac.PolicyRule {
+	rules := make([]rbac.PolicyRule, 0, len(in))
+	for _, rule := range in {
+		// We need to split this rule into multiple rules for RBAC
+		if isResourceRule(&rule) && isNonResourceRule(&rule) {
```

Comment:
Nit: This reads a bit strange but I can understand what it is doing. I had to read the definitions of the functions to get it though